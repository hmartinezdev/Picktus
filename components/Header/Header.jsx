import React, { Component } from 'react';
import Head from 'next/head';

class Header extends Component {
  render() {
    return (
      <Head>
        <title>Picktus</title>
        <link href="https://fonts.googleapis.com/css?family=Caveat" rel="stylesheet" />
        <meta name="viewport" content="initial-scale=1.0, width=device-width" />
      </Head>
    );
  }
}

export default Header;
