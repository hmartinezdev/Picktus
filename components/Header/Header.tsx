import React, { Component } from 'react';
import Head from 'next/head';

class Header extends Component {
  render() {
    return (
      <Head>
        <title>Picktus</title>
        <link href="https://fonts.googleapis.com/css?family=Josefin+Sans:400|Nanum+Pen+Script" rel="stylesheet" />
        <meta name="viewport" content="initial-scale=1.0, width=device-width" />
      </Head>
    );
  }
}

export default Header;
