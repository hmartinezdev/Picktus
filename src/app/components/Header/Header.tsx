import Head from 'next/head';
import React, { Component } from 'react';

class Header extends Component {
  public render(): React.ReactElement<Header> {
    return (
      <Head>
        <title>Picktus</title>
        <link href="https://fonts.googleapis.com/css?family=Josefin+Sans:400|Caveat" rel="stylesheet" />
        <meta name="viewport" content="initial-scale=1.0, width=device-width" />
      </Head>
    );
  }
}

export default Header;
