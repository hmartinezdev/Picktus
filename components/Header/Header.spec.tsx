import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

let wrapper;

const setup = () => shallow(<Header />);

describe('<Header />', () => {
  test('should render properly', () => {
    wrapper = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
