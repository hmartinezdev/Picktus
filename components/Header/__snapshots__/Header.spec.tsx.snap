// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render properly 1`] = `
<SideEffect(Head)>
  <title>
    Picktus
  </title>
  <link
    href="https://fonts.googleapis.com/css?family=Caveat"
    rel="stylesheet"
  />
  <meta
    content="initial-scale=1.0, width=device-width"
    name="viewport"
  />
</SideEffect(Head)>
`;
